src/message/ersip_hdr_refer_to.erl

```diff
@@ -34,9 +36,13 @@
 
 -spec new(ersip_uri:uri()) -> refer_to().
 new(URI) ->
-    #refer_to{display_name = {display_name, []},
+    new(URI, {display_name, []}).
+new(URI, DN) ->
+    new(URI, DN, ersip_hparams:new()).
+new(URI, DN, HParams) ->
```

Comment:
Please do not expose header params from header. This is helper for internal implementation.

Reply:
`new/3` removed